config/environments/test.rb

```diff
@@ -1,5 +1,6 @@
 Coaster::Application.configure do
   # Settings specified here will take precedence over those in config/application.rb
+  config.secret_token = '4038e98c8b1c1b101fc1cc78c7e14f9f7257517052fca5c1953d986d455ffdc0e5a313eb947da45ccc98fd1e75a1a8cebee4c69c3e5368ceb727d48d3592bf7f'
```

Comment:
Instead of setting a value for the test environment only, let's update the secret_token initializer to pull the value from Rails.application.secrets and set a test value in secrets.yml.example